test/router.spec.js

```diff
@@ -408,45 +419,57 @@ describe('createCentralRouter()', () => {
     describe('project viewer', () => {
       beforeEach(() => {
         mockLogin({ role: 'none' });
+      });
 
-        testData.extendedProjects.createPast(1, { role: 'manager', forms: 1 });
+      describe('ProjectSettings', () => {
```

Comment:
I noticed that most of the tests under `describe('project routes', ...)` weren't quite right: they were navigating to /projects/1/&hellip;, but then using the data from the second project. The tests of `ProjectSettings` are the only ones that need two projects, so I've split that off into its own `describe()` block. Each `describe()` block will create its own test data. That way, there won't be a need to change the tests of the other project routes.

Reply:
The tests under `describe('form routes', ...)` had a similar issue. That `describe()` block will now create its own test data.